clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/security/AzureNamedAccountCredentials.groovy

```diff
@@ -69,21 +73,20 @@ public class AzureNamedAccountCredentials implements AccountCredentials<AzureCre
 
   @Override
   String getName() {
-    return accountName
+    accountName
   }
 
   @Override
   public String getProvider() {
-    return getCloudProvider()
+    getCloudProvider()
   }
 
   private AzureCredentials buildCredentials() {
-
-    return new AzureCredentials(this.tenantId, this.clientId, this.appKey, this.subscriptionId)
+    new AzureCredentials(this.tenantId, this.clientId, this.appKey, this.subscriptionId)
   }
 
   private static List<AzureRegion> buildRegions(List<String> regions) {
-    return regions?.collect {new AzureRegion(it)}
+    regions?.collect {new AzureRegion(it)}
```

Comment:
Is it OK to return `null` here? Would `regions?.collect {new AzureRegion(it)} ?: []` be safer?